processor/pom.xml

```diff
@@ -92,6 +92,17 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestEntries>
+                            <MapStruct-Generated-Comment>${project.groupId}:${project.artifactId}:${project.version}</MapStruct-Generated-Comment>
```

Comment:
> So, we could then use: Implementation-Version

+1

> extended with the Build-Jdk

But that's the JDK the given MapStruct version itself was built with, right? I don't think that helps (similar to group/artifact id, we can find out that info just by examining the JAR). It would be very useful though to add the name + version of the compiler being used to compile the project MapStruct is applied to. I don't know though how that can be determined.